homeassistant/components/google_translate/tts.py

```diff
@@ -114,74 +123,14 @@ def supported_languages(self):
         """Return list of supported languages."""
         return SUPPORT_LANGUAGES
 
-    async def async_get_tts_audio(self, message, language, options=None):
+    def get_tts_audio(self, message, language, options=None):
         """Load TTS from google."""
+        tts = gTTS(text=message, lang=language)
+        mp3_data = BytesIO()
+
+        try:
+            tts.write_to_fp(mp3_data)
+        except gTTSError as exc:
+            _LOGGER.exception("Error during processing of TTS request %s", exc)
```

Comment:
```suggestion
            _LOGGER.exception("Error during processing of TTS request %s", exc)
            return None, None
```
We should return here, so it does not try to get the value of `mp3_data`